pre-commit-hooks/codespell_ignore.txt

```diff
@@ -0,0 +1,9 @@
+ser
+onWs
+datas
+AtLeast
+AtMost
+compiletime
+tring
+trings
```

Comment:
Where did this come from? I found it only in typos config.

Reply:
https://github.com/XRPLF/clio/blob/develop/src/rpc/JS.hpp#L24